scripts/is/intg/run-int-test.sh

```
@@ -0,0 +1,289 @@
+
+set -o xtrace
+
+TESTGRID_DIR=/opt/testgrid/workspace
+INFRA_JSON='infra.json'
+PRODUCT_REPOSITORY=$1
+PRODUCT_REPOSITORY_BRANCH=$2
+PRODUCT_NAME="wso2is"
+PRODUCT_VERSION=$4
+GIT_USER=$5
+GIT_PASS=$6
+TEST_MODE=$7
```

Shellcheck (0.11.0)

[warning] TEST_MODE appears unused. Verify use (or export if used externally).

(SC2034)